frontend/src/app/services/auth/auth.service.ts

```diff
@@ -5,106 +5,58 @@
-
-    if (!environment.production) {
-      this._accessToken =
-        this.localStorageService.getValue('access_token') || '';
-      this._refreshToken =
-        this.localStorageService.getValue('refresh_token') || '';
-    }
-  }
+  ) {}
 
-  get accessToken(): string {
-    return this._accessToken;
-  }
-
-  getRedirectURL(): Observable<GetRedirectURLResponse> {
-    return this.http.get<GetRedirectURLResponse>(
-      environment.backend_url + '/authentication',
-    );
+  getRedirectURL(): Observable<string> {
+    return this.http.get<string>(environment.backend_url + '/authentication');
   }
 
-  getAccessToken(code: string, state: string): Observable<PostTokenResponse> {
-    return this.http.post<PostTokenResponse>(
-      environment.backend_url + '/authentication/tokens',
-      {
+  getAccessToken(code: string, state: string): Observable<boolean> {
+    return this.http
+      .post<boolean>(environment.backend_url + '/authentication/tokens', {
         code,
         state,
-      },
-    );
-  }
-
-  performTokenRefresh(): Observable<RefreshTokenResponse> {
-    return this.http
-      .put<PostTokenResponse>(
-        environment.backend_url + '/authentication/tokens',
-        { refresh_token: this._refreshToken },
-      )
+      })
       .pipe(
-        tap({
-          next: (res) => this.logIn(res.access_token, res.refresh_token),
+        tap((_) => {
+          this.localStorageService.setValue('loggedIn', 'true');
         }),
       );
   }
 
+  performTokenRefresh(): Observable<RefreshTokenResponse> {
+    return this.http.put<PostTokenResponse>(
+      environment.backend_url + '/authentication/tokens',
+      {},
+    );
+  }
+
   isLoggedIn(): boolean {
-    return !!this._accessToken;
+    return this.localStorageService.getValue('loggedIn') === 'true';
   }
 
   webSSO() {
     this.getRedirectURL().subscribe((res) => {
-      window.location.href = res.auth_url;
-    });
-  }
-
-  logIn(accessToken: string, refreshToken: string) {
-    this._accessToken = accessToken;
-    this._refreshToken = refreshToken;
-    if (!environment.production) {
-      this.localStorageService.setValue('access_token', accessToken);
-      this.localStorageService.setValue('refresh_token', refreshToken);
-    }
-    this.cookieService.put('access_token', accessToken, {
-      path: '/prometheus',
-      sameSite: 'strict',
-    });
-    this.cookieService.put('access_token', accessToken, {
-      path: '/grafana',
-      sameSite: 'strict',
+      window.location.href = res;
     });
   }
 
   logOut() {
-    this._accessToken = '';
-    this._refreshToken = '';
-    if (!environment.production) {
-      this.localStorageService.setValue('access_token', '');
-      this.localStorageService.setValue('refresh_token', '');
-    }
     this.localStorageService.setValue('GUAC_AUTH', '');
-    this.cookieService.remove('access_token', { path: '/prometheus' });
+    this.localStorageService.setValue('loggedIn', '');
```

Comment:
```suggestion
    this.localStorageService.setValue('loggedIn', 'false');
```

loggedIn sounds more like a "boolean" value. I'd therefore also set it to false if not logged in.